### PR TITLE
Issue 3 user can save or remove recipes from bookmarked

### PR DESCRIPTION
**Pull request by CristoferMar**
CookItUp_Feature_3

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -58,6 +59,24 @@ $searchBar.addEventListener('submit', function (event) {
   responseGET(getRecipeIDs, 'filter.php?c=' + searched); // category
 });
 
+gallery.addEventListener('click', function (event) {
+  if (event.target.matches('.bookmark')) {
```
Comment (codingcodymiller) [inline js/main.js:63]: I would just move this conditional to the `handleBookmark` function as a guard so you can remove the need for an anonymous function here.

File: css/styles.css [hunk 2]
```diff
@@ -99,6 +99,11 @@ p {
 }
 
 /* Meal classes: */
+
+.Meals {
```
Comment (codingcodymiller) [inline css/styles.css:103]: You should avoid capitalizing class names like this.

File: css/styles.css [hunk 3]
```diff
@@ -114,6 +119,11 @@ p {
 }
 
 /* Drink classes: */
+
+.Drinks {
```
Comment (codingcodymiller) [inline css/styles.css:123]: You should avoid capitalizing class names like this.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (CristoferMar) [reply js/main.js:63]: Discussed with Cody to allow this formatting for future use of other events.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**